opentimelineio_contrib/adapters/tests/test_fcpx_adapter.py

```diff
@@ -1,7 +1,23 @@
 import os
+import subprocess
+import sys
 import unittest
 import opentimelineio as otio
 import opentimelineio.test_utils as otio_test_utils
+from opentimelineio_contrib.adapters.fcpx_xml import format_name
+
+try:
+    # Python 3.3 forward includes the mock module
+    from unittest import mock
+    could_import_mock = True
+except ImportError:
+    # Fallback for older python (not included in standard library)
+    try:
+        import mock
+        could_import_mock = True
+    except ImportError:
+        # Mock appears to not be installed
+        could_import_mock = False
```

Comment:
Note that the `mock` library is installed via one of the optional install targets in the setup.py.

Reply:
I basically replicated what you are is done in `test_common.py` here.

Reply:
Ah, and without that the testsuite was not passing on travis.